eng/pipelines/templates/jobs/ci.yml

```diff
@@ -68,6 +68,13 @@ jobs:
         displayName: 'Use Python 3.6'
         inputs:
           versionSpec: '3.6'
+      - task: securedevelopmentteam.vss-secure-development-tools.build-task-credscan.CredScan@2
```

Comment:
Do we also need to publish the results? https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/aggregate-reports.yml#L171